#This signal sends a 0 or 1 to the driver controller
int16 motor_left  #for the left motor
int16 motor_right #for the right motor